new-backend/server/apis/v1/services/informative.service.js

```diff
@@ -15,15 +16,29 @@ class InformativeService {
    * @returns {object} JSON representation of document
    * @memberof InformativeService
    */
-  async getByName(file) {
+  async getByName(folder, file) {
     try {
-      file += ".json";
-      // Open file containing our store
-      const pathToFile = path.join(process.cwd(), "App_Data/documents", file);
-      const text = await fs.promises.readFile(pathToFile, "utf-8");
-      // Parse the file content so we get an object
-      const json = await JSON.parse(text);
-      return json;
+      if (folder) {
+        file += ".json";
+        // Open file containing our store
+        const pathToFile = path.join(
+          process.cwd(),
+          "App_Data/documents/" + folder,
+          file
```

Comment:
I just noticed one more thing that I don't think has been mentioned before. Please use `path.join()` instead of assuming that `/` is the character that we should join folders by. The solution, as written now, is not OS-independent.

Reply:
Thank you for your review! We will look into this as soon as possible. We will get back to you about both V2 and path.join().